sdk/python/feast/ui_server.py

```diff
@@ -77,7 +77,7 @@ def read_registry():
     # For all other paths (such as paths that would otherwise be handled by react router), pass to React
     @app.api_route("/p/{path_name:path}", methods=["GET"])
     def catch_all():
-        filename = ui_dir + "index.html"
+        filename = str(ui_dir) + "/index.html"
```

Comment:
never really noticed, but the existing use of `ui_dir` _seems_ wrong given it's on that try resource block. Any ideas on how to change this?

Reply:
dear danny, I couldn't get you that point. existing ui_dir usage in whole file is working as expected without that line.